reactor-core/src/main/java/reactor/core/scheduler/WorkerTask.java

```diff
@@ -53,7 +66,9 @@
 	static final AtomicReferenceFieldUpdater<WorkerTask, Composite> PARENT =
 			AtomicReferenceFieldUpdater.newUpdater(WorkerTask.class, Composite.class, "parent");
 
-	Thread thread;
+	volatile Thread thread;
```

Comment:
@akarnokd is there a real need to pass this thread property volatile ?

Reply:
A lazySet is enough, that's what's happening with the field updater below.